benchmark/kostya/rapidjson_sax.h

```diff
@@ -15,10 +15,10 @@ struct rapidjson_sax {
         size_t k{0};
         double buffer[3];
         std::vector<point>* result;
-        
+
         Handler(std::vector<point> &r) { result=&r; }
 
-        bool Key(const char* key, SizeType length, bool copy) { 
+        bool Key(const char* key, SizeType length, bool copy)
```

Comment:
You just deleted the `{` character!!!! :-)

Reply:
Oh oops, thanks. fixing it